### ISSUE TITLE
BigInt literals are not detected

### ISSUE THREAD

**Issue by Apperk274** [typescript-rtti/typescript-rtti#106]
```ts
class Test {
  prop1: 123n
}

const prop1 = reflect(Test).getProperty('prop1')
prop1.type.isBigIntLiteral() // false
prop1.type.is('class') // true
```
